addOns/commonlib/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Maintenance changes involving commonlibs enchancement (Issue 6810) .
```

Comment:
There's a typo in enhancement. IMO this could just say `Maintenance changes (Issue 6810).`, the latter is implied by being in this add-on.